Parsers/ASimAuthentication/Parsers/ASimAuthenticationGithub.yml

```diff
@@ -0,0 +1,65 @@
+Parser:
```

Comment:
Auth parser for webhook is not created. please remove authentication related files

Reply:
Done